Dockerfile.template

```diff
@@ -89,7 +89,10 @@ RUN noInstallRecommends="" && \
 		vim \
 		wget \
 		zip && \
+	# get the semi-official latest-stable git instead of using the old(er) version from the ubuntu distro
 	add-apt-repository ppa:git-core/ppa && apt-get install -y git && \
+	# get the semi-official latest-stable git-lfs too
+	curl -s https://packagecloud.io/install/repositories/github/git-lfs/script.deb.sh | bash && apt-get install -y git-lfs && \
```

Comment:
This added line is the key change in this PR.
We add the packagecloud repo and then apt-get install git-lfs.

The only thing I'm not clear on is whether or not `$noInstallRecommends` should be provided as an argument to the apt-get command - that argument is used for most of the package installs, but the line that installs git does NOT include it. I followed git's example, but it might be that git's installation line is incorrect as it's inconsistent...

Reply:
LGTM